web/frontend/src/components/slam-2d-render.vue

```diff
@@ -99,13 +93,31 @@ renderer.setClearColor(color, 1);
 
 canvas.style.cssText = 'width:100%;height:100%;';
 
-const camera = new THREE.OrthographicCamera(-1, 1, 0.5, -0.5, -1, 1000);
-camera.userData.size = 2;
+const camera = new THREE.OrthographicCamera();
+camera.near = -1000;
+camera.far = 1000;
+camera.userData.size = 1;
 setCamera(camera);
 scene.add(camera);
 
+const baseMarker = makeMarker(BaseMarker, 'BaseMarker');
+const destMarker = makeMarker(DestMarker, 'DestinationMarker');
+destMarker.visible = false;
+
 const controls = new MapControls(camera, canvas);
 controls.enableRotate = false;
+controls.screenSpacePanning = true;
+
+const scaleObjects = () => {
+  const cameraScale = 25;
```

Comment:
Can we please have this as a constant at the top of the file?

Reply:
Done!